<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="filtersWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="WalletView">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Wallet"/>
    <addaction name="action_Open_Wallet"/>
    <addaction name="action_Save_Wallet"/>
    <addaction name="action_Save_Wallet_As"/>
    <addaction name="action_Delete_Wallet"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_Folder"/>
    <addaction name="action_Delete_Folder"/>
    <addaction name="separator"/>
    <addaction name="action_Add_Entry"/>
    <addaction name="action_Edit_Entry"/>
    <addaction name="action_Delete_Entry"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Website"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Set_Environment"/>
    <addaction name="action_Show_Passwords"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Launch_Domain"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="dataBaseTB">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New_Wallet"/>
   <addaction name="action_Open_Wallet"/>
   <addaction name="action_Save_Wallet"/>
   <addaction name="action_Delete_Wallet"/>
  </widget>
  <widget class="QToolBar" name="folderTB">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Add_Folder"/>
   <addaction name="action_Delete_Folder"/>
  </widget>
  <widget class="QToolBar" name="entriesTB">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_Add_Entry"/>
   <addaction name="action_Edit_Entry"/>
   <addaction name="action_Delete_Entry"/>
   <addaction name="separator"/>
   <addaction name="action_Show_Passwords"/>
   <addaction name="separator"/>
   <addaction name="action_Launch_Domain"/>
  </widget>
  <action name="action_New_Wallet">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/NewWallet.svg</normaloff>:/icons/NewWallet.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New Wallet</string>
   </property>
  </action>
  <action name="action_Open_Wallet">
   <property name="text">
    <string>&amp;Open Wallet</string>
   </property>
  </action>
  <action name="action_Save_Wallet">
   <property name="text">
    <string>&amp;Save Wallet</string>
   </property>
  </action>
  <action name="action_Save_Wallet_As">
   <property name="text">
    <string>&amp;Save Wallet As...</string>
   </property>
  </action>
  <action name="action_Delete_Wallet">
   <property name="text">
    <string>&amp;Delete Wallet</string>
   </property>
  </action>
  <action name="action_Add_Folder">
   <property name="text">
    <string>&amp;Add Folder</string>
   </property>
  </action>
  <action name="action_Delete_Folder">
   <property name="text">
    <string>&amp;Delete Folder</string>
   </property>
  </action>
  <action name="action_Website">
   <property name="text">
    <string>&amp;Website</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Set_Environment">
   <property name="text">
    <string>&amp;Set Environment</string>
   </property>
   <property name="toolTip">
    <string>Set Environment</string>
   </property>
  </action>
  <action name="action_Add_Entry">
   <property name="text">
    <string>Add Entry</string>
   </property>
  </action>
  <action name="action_Delete_Entry">
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </action>
  <action name="action_Launch_Domain">
   <property name="text">
    <string>&amp;Launch Domain</string>
   </property>
  </action>
  <action name="action_Show_Passwords">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Passwords</string>
   </property>
  </action>
  <action name="action_Edit_Entry">
   <property name="text">
    <string>Edit Entry</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
